ingestion_tools/scripts/gjensen_config.py

```diff
@@ -238,6 +238,7 @@ def to_standardization_config(
     "ts-tilt_range-min",
     "ts-tilt_range-max",
     "ts-total_flux",
+    "frame_dose_rate",
```

Comment:
I believe, the addition here will not have the desired impact as we haven't changed how we process for the frames entity.